deps/build.jl

```diff
@@ -50,4 +50,4 @@ provides(SimpleBuild,
              end)
           end), [libmumps_simple], os = :Unix)
 
-@BinDeps.install [:libmumps_simple => :libmumps_simple]
+@BinDeps.install Dict(:libmumps_simple => :libmumps_simple)
```

Comment:
I'm not sure we can be backward compatible with 0.3 since MPI.jl requires julia 0.4. Are you using MUMPS with juila 0.3?